sfepy/examples/diffusion/poisson_nonlinear_material.py

```diff
@@ -0,0 +1,108 @@
+r"""
+Nonlinear Poisson's equation example demonstrating 
+the nonlinear diffusion and nonlinear volume force terms.
+
+The example is an adaptation of:
+:ref:`diffusion-poisson_field_dependent_material`
+
+Find :math:`T` such that:
+
+.. math::
+   \int_{\Omega} c(T) \nabla s \cdot \nabla T + \int_{\Omega} g(T) \cdot s
+    = 0
+    \;, \quad \forall s \;.
+
+where :math:`c(T)` and :math:`g(T)`  are the :math:`T` dependent coefficients.
+"""
+from sfepy import data_dir
+from sfepy.base.base import output
+import numpy as np
+
+filename_mesh = data_dir + '/meshes/3d/cylinder.mesh'
+
+def get_conductivity(T):
+    """
+    Calculates the conductivity as (1 + T**4) and returns it.
+    """
+    val = 1 + T**4
+    return val
+   
```

Comment:
I can see an extra whitespace on this line. Maybe set your editor to mark or remove the trailing whitespace.